openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+    public isolated function init(ClientConfig clientConfig, string serviceUrl) returns error? {
+        http:ClientSecureSocket? secureSocketConfig = clientConfig?.secureSocketConfig;
+        http:Client httpEp = check new (serviceUrl, { auth: clientConfig.authConfig, secureSocket: secureSocketConfig });
+        self.clientEp = httpEp;
+    }
+    # Retrieves multiple records for the specified table.
+    #
+    # + tableName - Name of the table from which to retrieve the record.
+    # + nameValuePairs - Name-value pairs to use to filter the result set. This parameter is mutually exclusive with sysparm_query.
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields.
+    # + sysparmExcludeReferenceLink - Flag that indicates whether to exclude Table API links for reference fields
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmLimit - Maximum number of records to return. For requests that exceed this number of records, use the sysparm_offset parameter to paginate record retrieval. Note - Unusually large sysparm_limit values can impact system performance.
+    # + sysparmNoCount - Flag that indicates whether to execute a select count(*) query on the table to return the number of rows in the associated table.
+    # + sysparmOffset - Starting record index for which to begin retrieving records. Use this value to paginate record retrieval. This functionality enables the retrieval of all records, regardless of the number of records, in small manageable chunks.
+    # + sysparmQuery - Encoded query used to filter the result set. Syntax - sysparm_query=<col_name><operator><value>.
+    # + sysparmQueryCategory - Name of the category to use for queries.
+    # + sysparmQueryNoDomain - Flag that indicates whether to restrict the record search to only the domains for which the logged in user is configured.
+    # + sysparmSuppressPaginationHeader - Flag that indicates whether to remove the Link header from the response. The Link header provides various URLs to relative pages in the record set which you can use to paginate the returned record set.
+    # + sysparmView - UI view for which to render the data. Determines the fields returned in the response.
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object that includes all records of the table. Otherwise returns the relevant error.
+    remote isolated function getRecords(string tableName, string? nameValuePairs = (), string? sysparmDisplayValue = (), boolean? sysparmExcludeReferenceLink = (), string? sysparmFields = (), decimal? sysparmLimit = (), boolean? sysparmNoCount = (), decimal? sysparmOffset = (), string? sysparmQuery = (), string? sysparmQueryCategory = (), boolean? sysparmQueryNoDomain = (), boolean? sysparmSuppressPaginationHeader = (), string? sysparmView = (), string? accept = ()) returns json|error {
```

Comment:
The function name should be `getRecord` or `getRecordList` if result is an array.

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98